go/pkg/client/tree_test.go

```diff
@@ -1565,6 +1590,10 @@ func TestComputeMerkleTree(t *testing.T) {
 				t.Errorf("ComputeMerkleTree(...) = gave error %q, want success", err)
 			}
 			for _, ue := range inputs {
+				if ue.IsVirtualFile() {
+					gotBlobs[ue.Digest] = tc.digesttoBlob[ue.Digest]
```

Comment:
Why is this needed?

Reply:
chunker.New tries to:
 - read the content for blobs
 - read the file for files
 
Both of which won't be available in this case. So for testing purposes I'm just mapping the virtual inputs' digest back to the corresponding blob.

Reply:
Would it work if you excluded virtual lobs from `additionalBlobs` instead?

Reply:
yes, but I would still have to add a condition here since `ueFile` would not be (and probably should not be) handled in `chunker.New()`.

Reply:
That's what I had in mind. Thanks.